packages/gatsby/src/utils/source-nodes.ts

```diff
@@ -85,13 +85,16 @@ function deleteStaleNodes(state: IGatsbyState, nodes: Node[]): void {
 export default async ({
   webhookBody,
   parentSpan,
+  deferNodeMutation = false,
 }: {
   webhookBody?: unknown
   parentSpan?: Span
+  deferNodeMutation?: boolean
```

Comment:
Is this type signifying the signature of the function or the type of the local binding? If it's the latter, then it's not nullable since it defaults to boolean.